app/templates/experiments/info/results.hbs

```diff
@@ -1,2 +1,16 @@
 <h2>Responses</h2>
-{{experiment-results sessions=model serializeFields=serializeFields}}
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#participantData">Participant Data</a></li>
+  <li><a data-toggle="tab" href="#allData">All Data</a></li>
+</ul>
+
+<div class="tab-content">
+  <div id="participantData" class="tab-pane fade in active">
+    <h3>Participant Data</h3>
+    {{participant-data sessions=model}}
```

Comment:
Small regression: the profileIDs in the "participant data" tab are not being trimmed correctly. (compare table to JSON frame)

<img width="1116" alt="screen shot 2016-03-15 at 1 01 19 pm" src="https://cloud.githubusercontent.com/assets/2957073/13786482/1f5f6b18-eaae-11e5-89e9-1a52eff54a7a.png">